mapclientplugins/filelocationsinkstep/configuredialog.py

```diff
@@ -106,7 +106,7 @@ def getConfig(self):
         """
         self._previousIdentifier = self._ui.lineEdit0.text()
         config = {'identifier': self._ui.lineEdit0.text(),
-                  'file': PureWindowsPath(self._output_location()).as_posix(),
+                  'location': PureWindowsPath(self._output_location()).as_posix(),
```

Comment:
It would be better to use 'directory' here, there is already a concept for location at a higher level that is associated with the plugin.